### DISCUSSION TITLE
Radius influence with IconLayer

### DISCUSSION THREAD

**Discussion by sepatin** [visgl/deck.gl#5830] [Q&A]
Hi all, 

I have a newbie question : is there a simple way to manage a sphere of influence with an icon to get its pickedInfo ?
Indeed, i want to get picked info for my Icon depending on the mouse proximity rather than hovering precisely on it .

Thanks.
S.

**Comment by AdriSolid** [comment]
Try using `pickingRadius`, [check this out](https://deck.gl/docs/api-reference/core/deck#pickingradius)

**Reply by sepatin** [reply]
Thanks a lot & sorry for my leak of readings.

**Reply by AdriSolid** [reply]
No worries, glad it helped you!
